include/base.h

```diff
@@ -168,6 +168,7 @@ class reference
 	inline bool operator<(const reference<T>& other) const { return value < other.value; }
 	inline bool operator>(const reference<T>& other) const { return value > other.value; }
 	static inline void* operator new(size_t, void* m) { return m; }
+	static inline void* operator delete(void*);
```

Comment:
`include/base.h:171:43: error: ‘operator delete’ must return type ‘void’`

Reply:
@SadieCat,
Yes, just now I noticed my mistake.

Reply:
@SadieCat,
Fix it here:
https://github.com/inspircd/inspircd/pull/1879/commits/3a1b88a188604833350a6d8b87c081e5c396f0e8